Framework/CurveFitting/src/Algorithms/RefinePowderInstrumentParameters.cpp

```diff
@@ -483,10 +483,11 @@ void RefinePowderInstrumentParameters::refineInstrumentParametersMC(const TableW
  * Arguments
  * - fit2: boolean.  if True,then do Simplex fit for each step
  */
-void RefinePowderInstrumentParameters::doParameterSpaceRandomWalk(vector<string> parnames, vector<double> lowerbounds,
-                                                                  vector<double> upperbounds, vector<double> stepsizes,
-                                                                  size_t maxsteps, double stepsizescalefactor,
-                                                                  bool fit2) {
+void RefinePowderInstrumentParameters::doParameterSpaceRandomWalk(vector<string> &parnames,
```

Comment:
```suggestion
void RefinePowderInstrumentParameters::doParameterSpaceRandomWalk(const vector<string> &parnames,
```
Can this be `const` as well?